package.json

```diff
@@ -17,7 +17,8 @@
     "dev": "npm-run-all --parallel start watch",
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "lint": "tslint -p .",
+    "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
+    "lint-autofix": "eslint . -c .eslintrc.js --ext .ts,.tsx --fix",
```

Comment:
What about 
```suggestion
    "lint-autofix": "npm run lint -- --fix",
```
so we can share the same config

Reply:
anyway, you could run `yarn lint --fix` which is even less verbose than `yarn lint-autofix`

Reply:
I took these lines from another PR. Do we want to keep them coherent or try to improve them everywhere? In this case I suggest to skip the "autofix" it has broken more things than the ones it has fixed when I launched it.

Reply:
> try to improve them everywhere

the lesser the effort, the better ;)

>  it has broken more things than the ones it has fixed when I launched it.

that's weird, as autofix applied the very same rules defined for linting.

Reply:
It applied some "readonly"s that broke some code. By the way we can keep it and checking code after using it.

Reply:
Removed lint-autofix, you are right "yarn lint --fix" is less verbose.